acceptanceTests/setup.sh

```diff
@@ -27,44 +30,52 @@ echo "Starting minikube..."
 minikube start
 
 echo "Creating mizu tests namespaces"
-kubectl create namespace mizu-tests
-kubectl create namespace mizu-tests2
+kubectl create namespace mizu-tests --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
the idea is to not fail when the resource exists and if you do "kubectl create" it will write error. (not really fails)
but when you do dry run it returns the yaml and then you are running kubectl apply that knows to handle it and will "configure" the resource and as we already know the kubectl apply can handle upsert so it will create if not exists and update if exists

Reply:
if I'm not writing dry-run, it will create the namespace when I'm trying to get the yaml only without creation